### PR TITLE
Docker test rebase

### PR DESCRIPTION
**Pull request by davidslater**
Rebased off master with fixed test call.

### CODE REVIEW

**Comment by ng390** [comment]
Should we update the docs in contrib to reflect running pytest through the Docker containers?

**Comment by davidslater** [comment]
We should definitely document. However, I think we may need more discussion on how best to test everything framework related. My current thought is two suites of tests - one outside the docker container, which would run first, and a second suite that runs inside the docker container.

File: .github/workflows/ci_test.yml [hunk 1]
```diff
@@ -26,9 +26,10 @@ jobs:
       - name: Run python tests
         run: |
           pip install -r requirements.txt
+          version=$(python -m armory --version)
           bash docker/build-dev-minimal.sh
-          pip install -r test-requirements.txt
-          pytest -s --disable-warnings
+          docker run twosixarmory/tf1:${version} bash -c \
+          "cd /armory_dev; pytest -s --disable-warnings --ignore=tests/test_docker_management.py"
```
Comment (seanpmorgan) [inline .github/workflows/ci_test.yml]: nit: 
```suggestion
          docker run -w /armory_dev twosixarmory/tf1:${version} bash \
          pytest -s --disable-warnings --ignore=tests/test_docker_management.py
```

Comment (davidslater) [reply .github/workflows/ci_test.yml]: Is the "-w" run from a specific directory? I looked for a flag that did that, but the docker docs can be hard to parse.

Comment (seanpmorgan) [reply .github/workflows/ci_test.yml]: Yup, agree so many docs its confusing:
https://docs.docker.com/engine/reference/commandline/run/#options

Comment (davidslater) [reply .github/workflows/ci_test.yml]: Modified.

**Comment by davidslater** [comment]
@ng390 updated contributing
